internal/util/update/resource-merge.go

```diff
@@ -50,6 +52,16 @@ func (u ResourceMergeUpdater) Update(options UpdateOptions) error {
 	}
 	defer os.RemoveAll(updated.AbsPath())
 
+	err := settersutil.SetAllSetterDefinitions(
+		options.PackagePath,
```

Comment:
the latest shouldn't require both of these paths

Reply:
yes. Updated